symfony/plugins/orangehrmPimPlugin/entity/Employee.php

```diff
@@ -410,13 +427,12 @@ class Employee
      */
     private $workExperience;
 
-//    /**
-//     * @var Collection
-//     *
-//     * @ORM\ManyToMany(targetEntity="OrangeHRM\Entity\EmployeeEducation", mappedBy="employee")
-//     * @ORM\JoinColumn(name="emp_number", referencedColumnName="emp_number")
-//     */
-//    private $education;
+    /**
+     * @var Collection
+     *
+     * @ORM\OneToMany(targetEntity="OrangeHRM\Entity\EmployeeEducation", mappedBy="employee")
+     */
+    private $education;
```

Comment:
if this is a collection ==> `$educations` and in constructor initiate as ArrayCollection